tests/data/return_annotation_brackets.py

```diff
@@ -53,6 +53,27 @@ def foo(a: int, b: int, c: int,) -> intsdfsafafafdfdsasdfsfsdfasdfafdsafdfdsfasd
 def foo(a: int, b: int, c: int,) -> int:
     return 2
 
+# Deeply nested brackets
+# with *interesting* spacing
```

Comment:
love it

Reply:
I pray I never meet a developer who likes to write their brackets like these examples 😅

Reply:
You missed an opportunity to write ASCII art using weirdly spaced parentheses

Reply:
I've got a few sad faces `):` in there